frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -109,13 +115,18 @@ const ActivitySummary = ({
       <Helmet>
         <title>Activity summary</title>
       </Helmet>
+      <legend className="margin-top-1">
```

Comment:
Legend should be the first child of a ```<fieldset>``` element

https://html.spec.whatwg.org/multipage/form-elements.html#the-legend-element

Reply:
The way field sets are being used on this page for each item. So I think this would need to be made into a span.

Reply:
Yes, or a ```p class="usa-prose```, as long as it's not a legend

Reply:
Will update.